spec/counter_culture_spec.rb

```diff
@@ -1371,7 +1371,8 @@
     categ.reload.posts_count.should == 1
   end
 
-  describe "#previous_model" do
+  # TODO check that effect is tested rather than method directly
```

Comment:
I would want to keep this spec regardless. This method is supposed to implement specific behavior and I'd want to keep a test to ensure it actually works correctly, even if it's indirectly tested elsewhere.